tests/lex/test_lex.py

```diff
@@ -98,8 +98,14 @@ def test_lex_symbols():
 
 def test_lex_strings():
     """ Make sure that strings are valid expressions"""
-    objs = tokenize("\"foo\" ")
+    objs = tokenize('"foo"')
     assert objs == [HyString("foo")]
+    # See issue #831.
```

Comment:
I think this comment could be more descriptive, like `#  can tokenize (ignored) backslash-newline`. (Source comments occupy valuable real estate; dedicated future readers who want to find an associated ticket can always look for more info in the commit message, but casual readers might be annoyed by the exhortation to look up a magic number on GitHub.)